english/analyze/analysis.md

```diff
@@ -1,2 +1,2 @@
 Surprisingly, this is not really marshmallow. It only looks like it but in reality it is made out of... 
-Let me guess, your favorite flavor of marshmallow is avocado; mixed with some [coffee](../coffee/coffee.md)
\ No newline at end of file
+Let me guess, your favorite flavor of marshmallow is avocado and candy cane; mixed with some [coffee](../coffee/coffee.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.